src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -344,22 +403,31 @@ private void create()
             "feeLimit must be >= 0 and <= " + VMConfig.MAX_FEE_LIMIT);
       }
 
+      if (RuntimeImpl.IS_NEW_VERSION) {  // TODO according to version
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this trailing comment on the previous empty line. ([squid:TrailingCommentCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ATrailingCommentCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbTDq44O08i52vd9SHQ&open=AWbTDq44O08i52vd9SHQ)

[ISSUE_KEY]: AWbTDq44O08i52vd9SHQ